utils/gyb.py

```diff
@@ -304,7 +309,7 @@ def splitGybLines(sourceLines):
     dedents = 0
     try:
         for tokenKind, tokenText, tokenStart, (tokenEndLine, tokenEndCol), lineText \
-            in tokenize.generate_tokens(sourceLines.__iter__().next):
+            in tokenize.generate_tokens(partial(next, sourceLines.__iter__())):
```

Comment:
@frewsxcv unfortunately `next` only exists on an iterator in Python 2. [PEP 3114](https://www.python.org/dev/peps/pep-3114/) renamed the `next` function to `__next__` in Python 3 and got rid of `next` entirely. Or put another way, it works as expected in Python 2 and fails with the runtime error `AttributeError: 'list_iterator' object has no attribute 'next'` if you run the same code in Python 3.

Like I said before somewhere else, this is the first Python code I've ever written or worked with. I say that to say please forgive my ignorance when asking what may be a stupid question: what is a `lambda`? Is that some sort of Python keyword?